cmd/earth/main.go

```diff
@@ -1480,7 +1476,7 @@ func (app *earthApp) collectAnalytics(exitCode int, realtime time.Duration) {
 			Set("version", Version).
 			Set("gitsha", GitSha).
 			Set("exitcode", exitCode).
-			Set("ci", ci).
+			Set("ci", os.Getenv("CI")).
```

Comment:
BTW, I would also add a special case for Jenkins here.. (detect presence of `BUILD_ID` or something like that).

Also, I would make this explicitly `"false"` when the var doesn't exist, rather than `""` - it'll make filtering easier.

Reply:
ah that's a really good point. let me fix that.